.rhcicd/clowdapp-engine.yaml

```diff
@@ -213,6 +214,9 @@ parameters:
 - name: IMAGE
   description: Image URL
   value: quay.io/cloudservices/notifications-engine
+- name: IN_MEMORY_DB_ENABLED
+  description: If inMemoryDb is set to true, Clowder will pass configuration of an In Memory Database to the pods in the ClowdApp. This single instance will be shared between all apps.
```

Comment:
> This single instance will be shared between all apps.

Will be shared between all replicas from the  notifications-engine ClowdApp, or be shared across multiple ClowdApps?

Reply:
It was originally for a single ClowdApp, but it seems they've added support for sharing across ClowdApps. I need to read through [this Slack thread](https://redhat-internal.slack.com/archives/C022YV4E0NA/p1747311447205019?thread_ts=1740743811.150289&cid=C022YV4E0NA) and [RHCLOUD-40077](https://issues.redhat.com/browse/RHCLOUD-40077) to figure out what changes I need to make to support that.

Reply:
resolved, will leave as a per-app ClowdApp